autopilot/contractor.go

```diff
@@ -397,11 +402,6 @@ func (c *contractor) runContractFormations(ctx context.Context, cfg api.Autopilo
 		host := candidates[h]
 		hk := host.PublicKey
 
-		// break if the contractor was stopped
-		if c.isStopped() {
```

Comment:
why remove this?

Reply:
I moved an `isStopped` check to the beginning of all `runActionX` functions and removed them from the `for` loops. I don't really have a solid for removing it though I can revert it but it felt like we were doing `isStopped` checks all over the place.

I tried adding stop checks in place that take a while. Forming a couple of contracts does not take long at all, some with renewals

Reply:
will just re-add it's not worth discussing